src/Uno.UWP/System/DispatcherQueue.Android.cs

```diff
@@ -0,0 +1,22 @@
+﻿using Windows.UI.Core;
+
+namespace Windows.System
+{
+	partial class DispatcherQueue
+	{
+		bool TryEnqueueNative(DispatcherQueuePriority priority, DispatcherQueueHandler callback)
```

Comment:
Why do you need this

Reply:
DispatcherQueue is the new way to go in WinUI. All the code is using it internally. The current implementation is only a facade over the CoreDispatcher...  it sure needs to be updated in the future.